config/application.rb

```diff
@@ -8,6 +8,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(:default, Rails.env)
 
+ENV.update YAML.load(File.read('config/application.yml'))[Rails.env] rescue {}
```

Comment:
isn't better use (figaro)[https://github.com/laserlemon/figaro] ? because if you add circle ci, you can't upload this file